sklearn/tree/_export.py

```diff
@@ -569,6 +570,9 @@ def _make_tree(self, node_id, et, criterion, depth=0):
         return Tree(name, node_id, *children)
 
     def export(self, decision_tree, ax=None):
+        from sklearn.utils.validation import check_is_fitted
+        check_is_fitted(decision_tree)
```

Comment:
Why not using this at the very beginning of `plot_tree` instead?

Also it's already imported at the top of the file so no need to import again